proposals/20210917-mitre-tags.md

```diff
@@ -0,0 +1,12 @@
+# Add missing MITRE ATT&CK tags
+
+## Summary
+This is a proposal for adding missing MITRE ATT&CK tags to the existing Falco rules.
+
+We put a spreasheet together with our proposal at https://docs.google.com/spreadsheets/d/1lK2luf4iuOHDJEZ9EGcv3DkuuHeyRJAfeBzQCu2AuUI that we opened up for discussion during the community calls ([1](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-06-09), [2](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-06-23), [3](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-07-14), [4](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-07-21)).
+
+## Motivation
```

Comment:
Please add links to the new MITRE container framework.

Reply:
And you might want to explain what are the new tags you want to add in this proposal